Makefile.am

```diff
@@ -167,6 +167,7 @@ test_unit_uv_CFLAGS = $(AM_CFLAGS) -Wno-conversion
 test_unit_uv_LDADD = libtest.la
 
 test_integration_uv_SOURCES = \
+  ${libraft_la_SOURCES} \
```

Comment:
Looks unrelated to the rest of the commit, should this be split out?

Reply:
Those sources are needed for the calls to `UvBarrier` and `UvUnblock` in the new tests.